After transfer of repo. Modified references to new repo URL

hemulin opened the pull request.

hemulin committed:
After transfer of repo. Modified references to new repo URL

MichaelZelbel reviewed (approved):
LGTM

nourspace reviewed (approved):
The changes look good, but I wasn't able to understand why some of the CI jobs are failing.
Trying to build binaries from the branch to confirm.

nourspace commented:
Few suggestions
- I would close #1216 and only merge this one which is targeting the `main` branch
- It would be less confusing if we make a new release `release-v5.2.1` or even `release-v5.3.0`
  This would happen against the updated main branch as a release/tag not in a PR

nourspace commented:
I built the branch `hemulin:main` and was able to run my validator
<img width="971" alt="image" src="https://user-images.githubusercontent.com/1257310/209652351-f88c57e3-cf72-4001-9b56-a0753b598aae.png">

nourspace committed:
fix(ci): install libssl-dev as it seems missing for CI jobs

https://stackoverflow.com/questions/65553557/why-rust-is-failing-to-build-command-for-openssl-sys-v0-9-60-even-after-local-in

nourspace commented:
I created a batch to your main branch @hemulin 
https://github.com/hemulin/libra/pull/1

Please run the ci jobs and see whether the issue (installing openssl) is fixed. If so, you could merge the batch to your branch and hopefully the ci jobs run here as well.

We could also merge this PR and I would create another PR against the updated main. same same ^^

hemulin committed:
Merge pull request #1 from nourspace/patch-1

fix(ci): install libssl-dev as it seems missing for CI jobs
